src/lib/component/partial/Label/Label.js

```diff
@@ -31,33 +49,89 @@ const styles = theme => ({
   },
 });
 
+const imageExtensions = [
+  ".PNG",
+  ".JPG",
+  ".JPEG",
+  ".GIF",
+  ".WEBP",
+  ".TIFF",
+  ".PSD",
+  ".RAW",
+  ".BMP",
+  ".HEIF",
+  ".INDD",
+  ".SVG.",
+];
+
 class Label extends React.PureComponent {
   static propTypes = {
     classes: PropTypes.object.isRequired,
     name: PropTypes.string.isRequired,
     value: PropTypes.string.isRequired,
   };
 
-  parseLink = value => {
+  parseLink = (value, name) => {
     try {
       new URL(value);
+      // see if it's got an image extension
+      let results = imageExtensions.filter(ext =>
+        value.toUpperCase().includes(ext),
+      );
+      let imageMatch = results.length > 0;
+      // if that fails, we should check the content type in case
+      if (!imageMatch) {
+        fetch(value, {
+          method: "HEAD",
+        })
+          .then(response => response.headers.get("Content-type"))
+          .then(type => {
+            imageMatch = type === "image/jpeg" ? true : false;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation/List_of_default_Accept_values#Values_for_an_image

From the looks of it, most browsers are happy with anything that is `image/*`, so in theory we could do something like `/^image\/.+/.test(type)`